### DISCUSSION TITLE
Add option to disable "Save dashboard" modal

### DISCUSSION THREAD

**Discussion by mmcnl** [grafana/grafana#51214] [Dashboards, Panels & Visualizations]
First of  all, thank you for such an amazing tool and ecosystem! 🎉 

Our team is in the process of switching from Elastic Stack to Grafana and are very happy with it so far. However as I'm in the process of building dashboards and saving them frequently, I am seeing this "Save dashboard" modal very frequently
![image](https://user-images.githubusercontent.com/1498727/174916457-a761c104-a5ce-420a-ade3-3044c12554ae.png)
Since I am in prototyping mode right now, there is currently no need to leave comments about the changes and so I would prefer to just be able to make a single click on the save button and be finished (without the need to move the pointer over and click save on the modal).
![image](https://user-images.githubusercontent.com/1498727/174916594-df5d26be-3a57-4d6d-81eb-9e032cd4947c.png)

This issue seems to have been brought up before (https://github.com/grafana/grafana/issues/9604) but that issue was closed as completed without the issue seemingly actually being completed.

Many thanks for any insight or potential workarounds!

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
